API/.gitignore

```diff
@@ -0,0 +1,34 @@
+# compiled output
+/dist
+/node_modules
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+
+# OS
+.DS_Store
```

Comment:
can you move these OS & IDE specific things to the top level `.gitignore` file

Reply:
looks like these configs are in the top level now, and should be removed from this file :pray: